src/metrics_server/connection_metrics.spec.ts

```diff
@@ -49,10 +60,17 @@ const VALID_REPORT: HourlyConnectionMetricsReport = {
   userReports: [
     structuredClone(VALID_USER_REPORT),
     structuredClone(VALID_USER_REPORT2),
-    structuredClone(LEGACY_USER_REPORT),
+    structuredClone(LEGACY_COUNTRIES_USER_REPORT),
   ],
 };
 
+const LEGACY_REPORT: HourlyConnectionMetricsReport = {
+  serverId: 'legacy-id',
```

Comment:
serverId is not legacy yet. We use it to count servers. We should replace it with server time though.

Reply:
This doesn't say it's a legacy field. I just wanted to give the identifier a legacy prefix so the legacy report can be distinguished from the non-legacy report (vs non-meaningful 'id1' and 'id2').